### DISCUSSION TITLE
Anomoly Detection - Still having problems with ' Setting a random_state has no effect since shuffle is False. You should leave random_state to its default (None), or set shuffle=True.' #2603.

### DISCUSSION THREAD

**Discussion by DanFranSpearco** [pycaret/pycaret#2878] [Q&A]
I am trying to learn how to use pycaret for anomoly detection, and I continue to have problems with the part of the code:

`exp_ano101 = setup(data, normalize = True, 
                   ignore_features = ['MouseID'], 
                   session_id = 123)`

I get back the stated error above. I have gone through several forums to find the answer. I upgraded to the newest version of pycaret, downgraded my version of scikit-learn, and nothing has seemed to work. Are there any other solutions that I may be missing?

**Comment by AZFARHAD24511** [comment]
I regret to inform you that the pycaret library is not advised for use due to its significant number of bugs and the absence of support from official sources. After attempting to run their example with their dataset, I encountered numerous errors in Google Colab. These issues raise concerns about the reliability of pycaret, as it is unclear what processes are occurring behind the scenes.
for example try to do this example in google colab "[PyCaret 2 HousePrice Regresion.ipynb](https://github.com/pycaret/examples/blob/main/PyCaret_2_HousePrice_Regresion.ipynb)" and then you will face with so many **errors.** thats funny
